content/es/docs/reference/glossary/sysctl.md

```diff
@@ -0,0 +1,24 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  Una interfaz para obtener y asignar valores a parámetros del núcleo Unix activo.
+
+aka:
+tags:
+- tool
+---
+ `sysctl` es una interfaz común usadapara consultar o modificar artributos del 
+ núcleo Unix durante su ejecución.
+
+<!--more-->
+
+En los sistemas Unix-like, `sysctl` es el comando que usan los administradores,
+para ver o modificar esos valores y también el nombre de la llamada al sistema
+que realiza esta función.
+
+{{< glossary_tooltip text="Container" term_id="container" >}} runtimes y 
```

Comment:
```suggestion
{{< glossary_tooltip text="Contenedor" term_id="container" >}} runtimes y
```
maybe?
Also, “runtimes” is English and with no expansion into Spanish. Is that OK?

Reply:
I'm OK with the proposed change I only used "Container" after grepping the es/ dir and found  on  content/es/docs/concepts/architecture/nodes.md that used container runtime.
In technical Spanish literature is used untranslated, I've only seen translated on books published before 1990 and the translations would sound like a bad joke.

Reply:
Rethinking the paragraph, and with the help form the Wikipedia to eliminate any personal bias, I think that the new translation reflects better the intention of the English sentence:

"La ejecución del contenedor y de los complementos de red puede depender de los valores assignados via sysctl."